process-engine/process_engine_meta
# Add error persistance

Depends on:
* https://github.com/process-engine/process_engine_contracts/pull/55
* https://github.com/process-engine/flow_node_instance.repository.sequelize/pull/7
* https://github.com/process-engine/process_engine/pull/138

## What did you change?

* updated dependencies to use feature branches of `add_error_persistance`

## How can others test the changes?

In the script task tests, you can specifically run the error test by attaching an `only`-flag like so:
```
it.only('should throw an error when trying to execute a faulty script task', ...
``` 

If you run the tests with this change, you can inspect the database to check whether or not the state has been correctly persisted.
The table `FlowNodeInstances` of the postgres database should contain an entry for the script task with the following values:

* flowNodeId: Task_1qarghp
* state: 4
* error: ReferenceError: a is not defined
* isSuspended: FALSE

## PR-Checklist

Please check the boxes in this list after submitting your PR:
